### PR TITLE
[Gdrive] Fix: GDriveFiles deletion also deletes in GDriveFolders

### PR DESCRIPTION
**Pull request by philipperolet**
Description
---
Related to issue https://github.com/dust-tt/dust/issues/5100

When deleting a google drive file, we don't remove it from the GoogleDriveFolders table which can lead to unexpected issues

Risks
---
Check that the logic to delete both makes sense in all situations with @lasryaric

Deploy
---
Connectors

### CODE REVIEW

**Comment by philipperolet** [comment]
At the time I checked that all calls to deleteFile are either when we garbage collect, when we can't find the google drive file or when it has been trashed, so it seems legit to also delete it from google_drive_folders

File: connectors/src/connectors/google_drive/temporal/activities.ts [hunk 1]
```diff
@@ -847,6 +847,15 @@ async function deleteFile(googleDriveFile: GoogleDriveFiles) {
     const dataSourceConfig = dataSourceConfigFromConnector(connector);
     await deleteFromDataSource(dataSourceConfig, googleDriveFile.dustFileId);
   }
+  const folder = await GoogleDriveFolders.findOne({
+    where: {
+      connectorId: connectorId,
+      folderId: googleDriveFile.driveFileId,
+    },
+  });
+  if (folder) {
```
Comment (philipperolet) [inline connectors/src/connectors/google_drive/temporal/activities.ts:856]: is it worth making a transaction here? intuition: no, too edgy
but open to discussion

Comment (lasryaric) [reply connectors/src/connectors/google_drive/temporal/activities.ts:856]: I would indeed delete the GoogleDriveFolder and GoogleDriveFile in one transaction.

**Review by lasryaric — COMMENTED** [review]
Left one comment.

**Review by lasryaric — APPROVED** [review]
LGTM.

**Pull request merged**